src/main/cpp/t10_max_local.cpp

```diff
@@ -24,5 +24,23 @@
 using namespace std;
 
 int t10_max_local() {
-
-};
\ No newline at end of file
+	int a = 1;
+	int c = 0;
+	int e = 0;
+	int count = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'd' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4349150)